main/portable-scandir.c

```diff
@@ -0,0 +1,243 @@
+#include "general.h"
+/*
+ * Taken from https://raw.githubusercontent.com/ClusterLabs/pacemaker/master/replace/scandir.c
```

Comment:
Isn't it better to use non-raw URL, so that one can easily get the information (history, etc.) of the file?
https://github.com/ClusterLabs/pacemaker/blob/master/replace/scandir.c